b2sdk/bucket.py

```diff
@@ -419,6 +418,27 @@ def create_file(
         recommended_part_size=None,
         continue_large_file_id=None,
     ):
+        """
+        Creates a new file in this bucket using multiple remote od local sources.
+
+        Sources can overlap and remote sources would be prioretized overl local sources.
+        For more information and usage examples please see "Advanced usage" documentation.
+
+        :param list[b2sdk.v1.WriteIntent] write_intents: list of write intents (remote or local sources)
+        :param str new_file_name: file name of the new file
+        :param str,None content_type: content_type for the new file, if ``None`` content_type would be
+                        automatically determined from file name or it may be copied if it resolves
```

Comment:
won't it be automatically detected by the cloud based on the first 1kb? I think we should say that it will be automatically determined, period :)